<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Sklep</class>
 <widget class="QDialog" name="Sklep">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>730</width>
    <height>413</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>730</width>
    <height>413</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>730</width>
    <height>413</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>WyborSklepu</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#Sklep{

background-color:rgb(170, 0, 255);


}</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>390</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#pushButton
{
background-color :rgb(0, 170, 255);
color :black;
border-style: outset;
}</string>
   </property>
   <property name="text">
    <string>Logout</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>60</y>
     <width>341</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
#label{
color :rgb(0, 170, 255);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>100</y>
     <width>501</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#label_2
{
color :rgb(0, 170, 255);

}</string>
   </property>
   <property name="text">
    <string>Choose store and compare prices!</string>
   </property>
  </widget>
  <widget class="QPushButton" name="TescoButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">#TescoButton{

background-color :rgb(0, 170, 255);
color : black;
border-style: bold;
}
</string>
   </property>
   <property name="text">
    <string>Show items</string>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="LidlButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>240</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">#LidlButton{

background-color :rgb(0, 170, 255);
color : black;
border-style: outset;
}</string>
   </property>
   <property name="text">
    <string>Show items</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BiedronkaButton">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>240</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>SplitHCursor</cursorShape>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">#BiedronkaButton{

background-color :rgb(0, 170, 255);
color : black;
border-style: outset;
}</string>
   </property>
   <property name="text">
    <string>Show items</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>70</y>
     <width>341</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>380</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#pushButton_2
{
background-color :rgb(0, 170, 255);
color :black;
border-style: outset;
}</string>
   </property>
   <property name="text">
    <string>Change password</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>380</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#pushButton_3
{
background-color :rgb(0, 170, 255);
color :black;
border-style: outset;
}</string>
   </property>
   <property name="text">
    <string>Change email</string>
   </property>
  </widget>
  <widget class="QWidget" name="tescoImage" native="true">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>199</width>
     <height>79</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#tescoImage
{
border-image : url(:/obrazy/tesco.jpg);
}</string>
   </property>
  </widget>
  <widget class="QWidget" name="LidlImage" native="true">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>150</y>
     <width>199</width>
     <height>79</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#LidlImage
{
border-image : url(:/obrazy/lidl.jpg);
}</string>
   </property>
  </widget>
  <widget class="QWidget" name="BiedronkaImage" native="true">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>150</y>
     <width>199</width>
     <height>79</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#BiedronkaImage
{
border-image : url(:/obrazy/biedra.jpg);
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="SredniaTesco">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>295</y>
     <width>531</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="SredniaLidl">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>320</y>
     <width>531</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="SredniaBiedronka">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>350</y>
     <width>531</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
